packages/app-mobile/components/screens/ConfigScreen/plugins/PluginStates.search.test.tsx

```diff
@@ -2,7 +2,6 @@ import * as React from 'react';
 import { mockMobilePlatform, setupDatabaseAndSynchronizer, switchClient } from '@joplin/lib/testing/test-utils';
 
 import { render, screen, userEvent, waitFor } from '@testing-library/react-native';
-import '@testing-library/react-native/extend-expect';
```

Comment:
Jest matchers are [now extended by default and `extend-expect` imports can be removed](https://callstack.github.io/react-native-testing-library/docs/migration/v13#extend-jest-matchers-by-default).